modules/website/beaconApp/src/app/app.component.ts

```diff
@@ -1,7 +1,8 @@
 import { Component } from '@angular/core';
-import { OktaAuthService } from '@okta/okta-angular';
+import { OktaAuthStateService } from '@okta/okta-angular';
```

Comment:
it looks like this service has changed - so I basically disabled it throughout - as we are using the underlying okta library directly.. you'd need to work out how this component has changed in the okta-angular upgrade (in release notes probably?)